rclcpp/test/rclcpp/test_timer.cpp

```diff
@@ -319,6 +319,23 @@ TEST_P(TestTimer, test_failures_with_exceptions)
   }
 }
 
+/// Create a timer that starts as canceled
```

Comment:
It would be probably better to add the test case that

1. create canceled `timer`.
2. reset `timer`. (activate)
3. check timer callback is fired.